package.json

```diff
@@ -112,7 +112,7 @@
     "allure-commandline": "2.13.8",
     "classlist.js": "1.1.20150312",
     "core-js": "3.7.0",
-    "focus-trap": "6.1.4",
+    "focus-trap": "6.5.0",
```

Comment:
you need to update package-lock.json too

Reply:
Done.